runtime/jcl/common/system.c

```diff
@@ -70,12 +70,37 @@ jstring JNICALL Java_java_lang_System_getEncoding(JNIEnv *env, jclass clazz, jin
 	return getEncoding(env, encodingType);
 }
 
+#if JAVA_SPEC_VERSION >= 18
+jstring JNICALL Java_java_lang_System_getSysPropBeforePropertiesInitialized (JNIEnv *env, jclass clazz, jint sysPropID)
+{
+	const char *sysPropValue = NULL;
+	jstring result = NULL;
+	PORT_ACCESS_FROM_ENV(env);
+
+	switch (sysPropID) {
+	case 0:    /* os.version */
+		sysPropValue = j9sysinfo_get_OS_version();
+		if (NULL == sysPropValue) {
+			sysPropValue = "unknown";
+		}
```

Comment:
This is also missing the Windows-specific checking for an embedded space; see https://github.com/eclipse-openj9/openj9/blob/master/runtime/vm/vmprops.c#L821.

Reply:
I was assuming it was left out because it's 32-bit specific, but I see it's not 32-bit specific as `WIN32` is defined on 64-bit.

Reply:
Had same assumption, thanks catching that.
No local build for Windows specific code, need PR build verification.